packages/react-scripts/README.md

```diff
@@ -3,10 +3,10 @@
 ### **Important:** The currently supported version of **CRA** by `backpack-react-scripts` is up to `v4.0.3`. Versions above this may not work.
 
 This package is a fork of [Create React App](https://github.com/facebookincubator/create-react-app) (specifically the
-`react-scripts` package). It's intended to be used in conjuction with `create-react-app` like so:
+`react-scripts` package). It's intended to be used in conjuction with `create-react-app` like so (Ensure you are using Node.js 14+):
```

Comment:
Keep this as before.
Add a section before `This package is a fork of ...` to explain the changes:

> Node.js requirement - Because of the EOF for Node.js 12 and BRS has used a higher version of `node-sass`, you should make sure that your Node.js is 14+ (Better to use Node.js 16+).

> Script change - When executing the command `npx create-react-app my-app --scripts-version=@skyscanner/backpack-react-scripts --template @skyscanner/backpack --use-npm`, `npx` will install the latest `create-react-app` to your machine, and `create-react-app` will also check if you are using the latest `create-react-app` or not, if not, this command will break down and you can't create a project to use `BRS` as bundler, so for now you can't use commands like `npx create-react-app@4.0.3 my-app --scripts-version=@skyscanner/backpack-react-scripts --template @skyscanner/backpack --use-npm` to create a project, instead you should execute `npx create-react-app my-app --scripts-version=@skyscanner/backpack-react-scripts --template @skyscanner/backpack --use-npm`.